packages/plugins/src/cli/propose/index.ts

```diff
@@ -73,24 +74,15 @@ export const buildParsedConfigArray = async (
   scriptPath: string,
   isTestnet: boolean,
   sphinxPluginTypesInterface: ethers.Interface,
-  sphinxContext: SphinxContext,
+  foundryToml: FoundryToml,
+  getConfigArtifacts: GetConfigArtifacts,
```

Comment:
@RPate97 Fyi, I changed the params of this function slightly. Just wanted to lyk because I think you mentioned the tests in the website use this function

Reply:
I was referring to `buildDeploymentWithCompilerConfigs`, thanks though.